components/MenuManagerBundle/src/lib/Service/DataTransformer/MenuItemsCollectionTransformer.php

```diff
@@ -32,10 +32,13 @@ public function __construct(MenuItemConverter $menuItemConverter)
      * Transforms a FieldType Value into a hash using `FieldTpe::toHash()`.
      * This hash is compatible with `reverseTransform()`.
      *
-     * @return array|null the value's hash, or null if $value was not a FieldType Value
+     * @return string|false the value's hash, or null if $value was not a FieldType Value
+     * @throws UnexpectedTypeException
      */
-    public function transform($value)
+    public function transform($value): bool|string
     {
+        dump($value); // ici on a un Doctrine\ORM\PersistentCollection
+        dump($value->getValues()); // ERROR 500 TypeError ReflectionProperty::getValue(): Argument #1 ($object) must be provided for instance properties
```

Comment:
dump($value); // ici on a un Doctrine\ORM\PersistentCollection
        dump($value->getValues()); // ERROR 500 TypeError ReflectionProperty::getValue(): Argument #1 ($object) must be provided for instance properties

Reply:
@erdnaxelaweb C'est là le problème.

Reply:
error fixed by https://github.com/Novactive/Nova-eZPlatform-Bundles/commit/0f2c5bdc30878d5d8eaf515a5eb29fec5ff9fb55